.gitignore

```diff
@@ -7,7 +7,16 @@ local.properties
 .idea/
 *.iml
 
+# Eclipse
```

Comment:
Not sure we should keep it for Studio-based project, but if it is more comfortable for you — not a problem.